lxd/storage/drivers/driver_zfs.go

```diff
@@ -68,6 +68,10 @@ func (d *zfs) load() error {
 	for _, tool := range []string{"zpool", "zfs"} {
 		_, err := exec.LookPath(tool)
 		if err != nil {
+			if shared.InSnap() {
+				return fmt.Errorf("Required tool '%s' is missing, consider installing ZFS tools in the host and use 'snap set lxd zfs.external=true'", tool)
```

Comment:
Please use %q to double quote placeholder strings.

Also can we is IsSnap() to alter message to say something about the host kernel version not matching the bundle zfs tooling?

Reply:
Let me know what you think of the reworded error. Thanks